docs/articles/intro/create-a-swf-deployment.rst

```diff
@@ -128,4 +133,5 @@ The content of the files can be copied and adapted from the PELUX Baseline
 Software Factory which is one such SWF Deployment. The README.md should
 contain a description on how to build the project with CMake.
 
+.. _`Git workflow`: http://pelux.io/software-factory/master/swf-blueprint/docs/articles/workflow/git-workflow.html#general-setup
```

Comment:
I'm wondering if the might be another way to refer to a chapter within the documentation. I think there is some way of tagging and referring to the tag.

Reply:
Yes. Will fix it now